SDL/SDLMain.cpp

```diff
@@ -768,6 +768,7 @@ int main(int argc, char *argv[]) {
 #if !PPSSPP_PLATFORM(SWITCH)
 	} else if (g_Config.iGPUBackend == (int)GPUBackend::VULKAN) {
 		SDLVulkanGraphicsContext *ctx = new SDLVulkanGraphicsContext();
+		mode |= SDL_WINDOW_VULKAN;
 		if (!ctx->Init(window, x, y, mode, &error_message)) {
 			printf("Vulkan init error '%s' - falling back to GL\n", error_message.c_str());
 			g_Config.iGPUBackend = (int)GPUBackend::OPENGL;
```

Comment:
> 

SDL_WINDOW_VULKAN  It must be in ". Init(window, x, y, mode, &error_message) "before, otherwise it will crash in the sdl library, I'm not sure why

Reply:
Oh, my mistake - it's passed in as a parameter.  Can we use `if (!ctx->Init(window, x, y, mode | SDL_WINDOW_VULKAN, &error_message)) {` then?  I just want to ensure the GL init that's run when Vulkan fails works properly.

-[Unknown]